cmake/common.cmake

```diff
@@ -312,13 +312,16 @@ if(COMPILE_TARGET STREQUAL "sgx")
     DESTINATION lib
   )
 endif()
+
 add_library(http_parser.host "${HTTP_PARSER_SOURCES}")
 set_property(TARGET http_parser.host PROPERTY POSITION_INDEPENDENT_CODE ON)
-install(
-  TARGETS http_parser.host
-  EXPORT ccf
-  DESTINATION lib
-)
+if(NOT COMPILE_TARGET STREQUAL "sgx")
```

Comment:
Would it be more precise to state which platforms do need this rather than which ones don't?

Reply:
I'd also rather have a INSTALL_HOST_LIBRARIES, set in a single place if we can.